libs/cardano-ledger-core/src/Cardano/Ledger/BaseTypes.hs

```diff
@@ -632,16 +650,17 @@ instance NoThunks ActiveSlotCoeff
 
 instance NFData ActiveSlotCoeff
 
-instance DecCBOR ActiveSlotCoeff where
-  decCBOR = mkActiveSlotCoeff <$> decCBOR
+instance FromCBOR ActiveSlotCoeff where
+  fromCBOR = mkActiveSlotCoeff <$> fromCBOR
+
+instance ToCBOR ActiveSlotCoeff where
+  toCBOR x@(ActiveSlotCoeff _ _) =
+    let ActiveSlotCoeff {..} = x
+     in toCBOR unActiveSlotVal
```

Comment:
With previous implementation it was obvious that a field was ignored. Current implementation makes it totally not obvious, therefore a comment is a must
```suggestion
     -- `unActiveSlotLog` is not encoded, since it can be derived from `unActiveSlotVal`
     in toCBOR unActiveSlotVal
```